[2024-12-03T14:34:53.723+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-03T14:34:53.734+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data manual__2024-12-03T14:34:44.263281+00:00 [queued]>
[2024-12-03T14:34:53.740+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data manual__2024-12-03T14:34:44.263281+00:00 [queued]>
[2024-12-03T14:34:53.741+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-03T14:34:53.756+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-03 14:34:44.263281+00:00
[2024-12-03T14:34:53.765+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=104) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-03T14:34:53.767+0000] {standard_task_runner.py:72} INFO - Started process 106 to run task
[2024-12-03T14:34:53.767+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb', 'load_data', 'manual__2024-12-03T14:34:44.263281+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb.py', '--cfg-path', '/tmp/tmpgjspblx8']
[2024-12-03T14:34:53.768+0000] {standard_task_runner.py:105} INFO - Job 186: Subtask load_data
[2024-12-03T14:34:54.015+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb.load_data manual__2024-12-03T14:34:44.263281+00:00 [running]> on host 58140385b9ff
[2024-12-03T14:34:54.077+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T14:34:44.263281+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T14:34:44.263281+00:00'
[2024-12-03T14:34:54.078+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-03T14:34:54.078+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-03T14:34:54.078+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-12-03 14:34:53.735270+00:00
[2024-12-03T14:34:54.079+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb and current dag run status:running
[2024-12-03T14:34:54.079+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-03T14:34:54.148+0000] {logging_mixin.py:190} INFO - Données chargées dans MongoDB, collection 'cleaned_data'
[2024-12-03T14:34:54.149+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-03T14:34:54.157+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-03T14:34:54.158+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb, task_id=load_data, run_id=manual__2024-12-03T14:34:44.263281+00:00, execution_date=20241203T143444, start_date=20241203T143453, end_date=20241203T143454
[2024-12-03T14:34:54.167+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-03T14:34:54.168+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-03T14:34:54.168+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb queued_at:2024-12-03 14:34:44.282284+00:00
[2024-12-03T14:34:54.169+0000] {logging_mixin.py:190} INFO - Task hostname:58140385b9ff operator:PythonOperator
[2024-12-03T14:34:54.183+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-03T14:34:54.347+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-03T14:34:54.349+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
